doc/decisions/decision_process.md

```diff
@@ -64,6 +72,8 @@ We base our decision process and template on:
 Following subsections describe all steps a decision might run through.
 Only two of them are mandatory.
 
+In all steps we avoid discussions in issues or PRs but directly update the decision text with the different opinions and discuss the text of the decision PR.
```

Comment:
> At least in early and volatile stages of the decision process, this approach sounds like generating quite some overhead.

Yes, I clarified: cb5b9129a68f0302894ed500a6c6bb1dbf11ad59

> 1.1. Maybe brainstorming techniques/sessions can help with finding more options.

If we (or someone) totally gets stuck, we might do this in a meeting. It never happened on a technical topic, though. So I didn't mention it. Furthermore, a meeting protocol would be needed, so that nothing important gets lost for the decision. Please give a concrete suggestion in the decision text about what would help you.